examples/references-count/src/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+  <body>
+    <script type="text/javascript" src="./index.ts"></script>
```

Comment:
Does it make sense to show a small info text here to indicate that this will never show up ?

Reply:
I don't know, does it? 
Do you mean the comment or actual DOM node, like `<p>` with the message?

Reply:
Yes, maybe just a small text saying that this app should only be used within contentful in the entry list location. 
Just for people that are running it locally and want to visit it in the browser, then they know it is not broken.

Reply:
Agree, done ✅